.github/workflows/docs.yml

```diff
@@ -6,7 +6,7 @@ permissions:
 on:
   workflow_dispatch:
   push:
-    # GONE FROM THIS FORK:
+    # NOW REMOVED FROM THIS FORK:
```

Comment:
Special case for updates I think should be done in *this* file - `.github/workflows/docs.yml`:

```diff
-    # NOW REMOVED FROM THIS FORK:
+    # [REMOVED FROM THIS FORK]
```